modules/documentation/slides_documentation.md

```diff
@@ -0,0 +1,57 @@
+<!-- .slide: data-state="title" -->
+
+# Documentation
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## What is documentation?
+
+- Provides context for your work
+- Allows your collaborators and future you to understand what has been done and why
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## Why document software?
+
+Make your software reusable:
+ 
+- A user should be able to run your software in the same way as you do now 
+- A user should be able to install your software
+- A contributor should be able to add or improve code
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## Documentation types
+
+User documentation
+- Purpose: What does the software do?
+- How can it be used: provide examples
+
+Developer documentation
+- How can your software be modified or extended?
+
+Deployment documentation
+- Hardware and software requirements
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## Documentation formats
+
+<!-- add visual with documentation format with increasing complexity -->
+
+- In-code: intended for contributors (comments, docstrings, ...)
+- README: simple text file, first thing that users/collaborators see
+- Websites, Wikis: 
+
```

Comment:
There are a few other aspects that are covered in the [project documentation part of the chapter](https://github.com/c-martinez/the-turing-way/pull/3/files?short_path=75ced25#diff-9fa24c5632919370a21ebec6d53852f6fca958681364e0cdfc9adf3d139e2f36): contribution guidelines, roadmap, changelog, code of conduct.

This is probably not applicable to every type of software (e.g. not for small visualization scripts), but maybe it is worth mentioning that it may be useful depending on your type of software (e.g. libraries).